architecture/core.rst

```diff
@@ -415,8 +415,11 @@ kernel/interrupt.h_. Interrupt consumers main API functions are:
       enable or disable an interrupt.
 
     - ``interrupt_mask()`` and ``interrupt_unmask()`` to respectively
-      mask or unmask an interrupt. Masking an interrupt keeps it enabled
-      but disables the asynchronus notification of the event.
+      mask or unmask an interrupt.
+      Masking of an enabled interrupt temporarily disables the interrupt
+      while unmasking enables a previously masked interrupt. interrpt_mask()
```

Comment:
It seems that you copied my typos. s/interrpt_/interrupt_/ on this line and on the line below